nicegui/helpers.py

```diff
@@ -14,7 +14,7 @@ def is_coroutine(object: Any) -> bool:
     return asyncio.iscoroutinefunction(object)
 
 
-def safe_invoke(func: Union[Callable, Awaitable], client: Optional[Client] = None) -> None:
+def safe_invoke(func: Union[Callable, Awaitable], client: Optional[Client] = None, *args: List[Any]) -> None:
```

Comment:
"For variable positional arguments (*args) and variable keyword arguments (**kw) you only need to specify the expected value for one such argument."
https://stackoverflow.com/a/37032111/3419103
https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values

Reply:
Thanks! I changed the code a bit to make the client parameter optional.